app/assets/stylesheets/common/components/color-input.scss

```diff
@@ -0,0 +1,23 @@
+.color-picker {
+  display: flex;
+  * {
```

Comment:
Any way to avoid this? I would imagine the height being fixed would cause problems when zooming in/out.

Reply:
100% zoom
![image](https://user-images.githubusercontent.com/1385470/151831261-b63c0da7-b2c9-4df6-ab0c-bf7b23747af4.png)

150% zoom
![image](https://user-images.githubusercontent.com/1385470/151831314-dd399b60-9de1-4113-8dd6-6cb4895cfae3.png)

I don't see any immediate problem, but I'd be happy to make it more robust if you have any suggestion.

Reply:
You're right, there is no immediate problem. We usually avoid setting heights on input elements, but in this case it makes sense, because the `type="color"` inputs are a bit special, they need padding to be reset.